docs-app/postcss.config.js

```diff
@@ -0,0 +1,17 @@
+const env = process.env.EMBER_ENV || 'development';
+
+const plugins = [require('autoprefixer')];
+
+if (env === 'production') {
+  /*
+  plugins.push(
+    require('cssnano')({
+      preset: 'default',
+    })
+  );
+  */
```

Comment:
@buschtoens found that `cssnano` may not be needed anymore, because Embroider handles minifying CSS in production.

https://github.com/embroider-build/embroider/blob/v1.8.3/packages/webpack/src/ember-webpack.ts#L332-L340